modules/tinymce/src/core/test/ts/webdriver/paste/CopyAndPaste.ts

```diff
@@ -0,0 +1,143 @@
+import { RealClipboard, RealMouse } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyApis, TinyAssertions, TinyHooks, TinyUi } from '@ephox/mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+const hook = TinyHooks.bddSetup<Editor>({
+  base_url: '/project/tinymce/js/tinymce',
+  toolbar: false,
+  statusbar: false
+}, []);
+
+type Selection = [ number[], number, number[], number ];
+
+const copyAndPast = async (editor: Editor, source: Selection, target: Selection): Promise<void> => {
+  const ui = TinyUi(editor);
+  const api = TinyApis(editor);
+
+  api.setSelection(...source);
+  ui.clickOnMenu('button:contains("Edit")');
+  await ui.pWaitForUi('*[role="menu"]');
+  await RealMouse.pClickOn('div[title="Copy"]');
```

Comment:
Why is this needed? Shouldn't this just be able to be `RealClipboard.pCopy()`?

Reply:
replacing with it seems not to trigger the copy, is that possible that I'm using the wrong `selector`, I tried with `RealClipboard.pCopy('iframe => body')`, but it does not work, so I may have misunderstood how selectors work in those functions

Reply:
sorry I forgot a detail about this one, in reality copy event is triggered (I'm checking with `editor.on('copy', (e) => console.log('copy: ', e));`), but the content seems not copied in the clipboard, using:
```
yarn bedrock-auto -b chrome -f modules/tinymce/src/core/test/ts/webdriver/paste/CopyAndPasteTest.ts
```
I get as a result of the `paste` the last thing that I copy in my clipboard and using `chrome-headless` I get nothing

Reply:
`RealClipboard.pCopy('iframe => body')` should work in this case... oh well, if you've tried it and it's not working then I'm happy with this workaround.

Reply:
Ahh I just looked up the test cases for this API and it has this which might explain why that didn't work: https://github.com/tinymce/tinymce/blob/674a17a92c2f91e8ca9af4e6abd188e8993f5fdd/modules/agar/src/test/ts/webdriver/RealEffectsTest.ts#L21

Reply:
ok, I added a comment, if it is ok, I'll resolve the conversation